skill_analysis.ipynb

```diff
@@ -114,44 +116,44 @@
     "\n",
     "# Change Assistant API version if needed\n",
     "# Find Latest --> https://cloud.ibm.com/docs/services/assistant?topic=assistant-release-notes\n",
-    "API_VERSION = '2019-02-28'\n",
-    "\n",
-    "# Change URL based on IBM Cloud datacenter you use   \n",
-    "URL = \"https://gateway.watsonplatform.net/assistant/api\" # Dallas (Default US South)\n",
-    "\n",
-    "#URL = \"https://gateway-s.watsonplatform.net/assistant/api\" # Dallas Staging\n",
-    "#URL = \"https://gateway-wdc.watsonplatform.net/assistant/api\" # Washington, DC\n",
-    "#URL = \"https://gateway-fra.watsonplatform.net/assistant/api\" # Frankfurt\n",
-    "#URL = \"https://gateway-syd.watsonplatform.net/assistant/api\" # Sydney\n",
-    "#URL = \"https://gateway-tok.watsonplatform.net/assistant/api\" # Tokyo\n",
-    "#URL = \"https://gateway-lon.watsonplatform.net/assistant/api\" # London\n",
+    "API_VERSION = '2020-04-01'\n",
+    "\n",
+    "# choose a datacenter to use\n",
+    "datacenters = {\n",
+    "    'dallas'    : ('https://gateway.watsonplatform.net/assistant/api',         'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'washington': ('https://gateway-wdc.watsonplatform.net/assistant/api',     'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'frankfurt' : ('https://gateway-fra.watsonplatform.net/assistant/api',     'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'sydney'    : ('https://gateway-syd.watsonplatform.net/assistant/api',     'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'tokyo'     : ('https://gateway-tok.watsonplatform.net/assistant/api',     'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'london'    : ('https://gateway-lon.watsonplatform.net/assistant/api',     'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'staging'   : ('https://gateway-s.watsonplatform.net/assistant/api',       'https://iam.cloud.ibm.com/identity/token'),\n",
+    "    'internal'  : ('https://api.us-south.assistant.test.watson.cloud.ibm.com', 'https://iam.test.cloud.ibm.com/identity/token')\n",
+    "}\n",
+    "URL, authenticator_url = datacenters['dallas']\n",
     "\n",
     "# For ICP(IBM Cloud Private), you can disable SSL verification by changing this to True\n",
     "DISABLE_SSL_VERTIFICATION = False \n",
     "\n",
-    "# By default we only need the IAM API Key & the Workspace ID\n",
+    "# OPTIMALLY, we authenticate via an iam api key\n",
```

Comment:
we still want to mention workspace id here as part of the description since is required. this notebook is trying to target a very board audience. there are people who are less familiar with python and WA but still want to use our notebook.